src/Components/addItem.js

```diff
@@ -23,21 +28,54 @@ class AddItem extends React.Component {
       purchaseFrequency: e.target.value,
     });
   };
+
+  onClose = () => {
+    this.setState({
+      show: !this.state.show,
+    });
+  };
+
   /* user is placeholder for user token when functionality is completed */
   addItem = e => {
     e.preventDefault();
     const db = fb.firestore();
-    db.collection(getLocalToken()).add({
-      itemName: this.state.itemName,
-      purchaseFrequency: this.state.purchaseFrequency,
-      lastPurchasedDate: this.state.lastPurchasedDate,
-    });
+    const collection = db.collection(getLocalToken());
 
-    this.setState({
-      itemName: '',
-      purchaseFrequency: null,
-      lastPurchasedDate: null,
-    });
+    var query = collection.where('itemName', '==', this.state.itemName);
+
+    query
+      .get()
+      .then(querySnapshot => {
+        if (querySnapshot.size) {
+          console.log('this name is already in db:', this.state.itemName);
```

Comment:
Debug statements like this are best not committed to the codebase, since our users will also see them pop up in their terminal. 😄